dashboard/dashboard.py

```diff
@@ -437,11 +438,31 @@ def display_apod() -> None:
 def display_neos() -> None:
     """Displays the day's Near-Earth objects and their associated metrics"""
 
+    neo_params = {
+        "start_date": TODAY,
+        "end_date": TODAY,
+        "api_key": API_KEY
+    }
+    data = get_neos(NEO_URL, neo_params, TODAY)
+    st.metric(f"Number of NEOs Today", len(data), border=True)
```

Comment:
quibble: no need for interpolation on a string with no variables
```suggestion
    st.metric("Number of NEOs Today", len(data), border=True)
```